SPECS/libcontainers-common/libcontainers-common.spec

```diff
@@ -54,6 +54,7 @@ Patch3:         CVE-2024-3727.patch
 #Note (mfrw): The patch for CVE-2024-37298 only applies to podman.
 Patch4:         CVE-2024-37298.patch
 Patch5:         CVE-2024-6104.patch
+Patch6:         CVE-2024-9407-01.patch
```

Comment:
Why -01 is there a -02 to accompany this?

Reply:
I was going to add a patch 02 before we determined this package didn't need to be patched at all.